api/src/opentrons/protocol_engine/state/state_summary.py

```diff
@@ -34,3 +35,4 @@ class StateSummary(BaseModel):
     wells: List[WellInfoSummary] = Field(default_factory=list)
     files: List[str] = Field(default_factory=list)
     liquidClasses: List[LiquidClassRecordWithId] = Field(default_factory=list)
+    tasks: Dict[str, TaskSummary] = Field(default_factory=dict)
```

Comment:
Ideally would be a list as mentioned in previous comments
```suggestion
    tasks: List[TaskSummary] = Field(default_factory=list)
```